sdk/src/test/tests/TestManagedNativeClass.ls

```diff
@@ -85,69 +83,137 @@ final native class MyChildManagedNativeClass extends MyManagedNativeClass {
     {
         super.intProperty = value + 2;
     }
-
-    
 }
 
-class TestManagedNativeClass extends LegacyTest
+class TestManagedNativeClass
 {
     function retrieveChild(owner:MyManagedNativeClass) {
         
-        // get the child, and make sure the script value is still valid
-        // magic! 
+        // get the child, and make sure the script value is still valid magic! 
         var child = owner.getChild();
-        log(child.scriptString);
-        log((child as MyChildManagedNativeClass).scriptStringChild);
+        Assert.equal(child.scriptString, "Hello, ");
```

Comment:
`Assert.compare` might be more appropriate for this one and the ones below. Functionally the same, but semantically different as `compare` is used to compare the runtime result value with a predefined expected value (and the error message on failure reflects that) whereas equals is just a generic equality check.